<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="secTypeBox">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="secTypeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="pageBox">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionBasicInfo"/>
    <addaction name="actionContractDetails"/>
    <addaction name="actionFundamentals"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionTable"/>
    <addaction name="actionChart"/>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menuHistory"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBasicInfo">
   <property name="text">
    <string>Basic Info</string>
   </property>
  </action>
  <action name="actionContractDetails">
   <property name="text">
    <string>ContractDetails</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionChart">
   <property name="text">
    <string>Chart</string>
   </property>
  </action>
  <action name="actionFundamentals">
   <property name="text">
    <string>Fundamentals</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
